lrauv_ignition_plugins/test/helper/LrauvTestFixture.hh

```diff
@@ -302,4 +303,15 @@ class LrauvTestFixture : public ::testing::Test
   /// \brief Publishes commands
   public: ignition::transport::Node::Publisher commandPub;
 };
+
+
+/// \brief Loads the default "buyant_tethys.sdf" world.
+class LrauvTestFixture : public LrauvTestFixtureBase
+{
+  /// Documentation inherited
```

Comment:
```suggestion
  // Documentation inherited
```

So, within the Ignition codebase, we leave "Documentation inherited" comments to explain that we're not adding documentation on purpose. When it's not added, the documentation is taken from the base class. But if the comment is added with three `///`, then that actually overrides the base class' doc block and writes "Documentation inherited" as the documentation.

Not important here since we're not generating docs for these classes, but I thought I should mention